designer/client/components/graph/node-modal/properties.tsx

```diff
@@ -94,10 +97,15 @@ export function Properties({
                 node={node}
                 renderFieldLabel={renderFieldLabel}
                 setProperty={setProperty}
-                fieldType={FieldType.checkbox}
-                fieldLabel={"Spill state to disk"}
-                fieldProperty={"typeSpecificProperties.spillStateToDisk"}
-                validators={[errorValidator(fieldErrors || [], "spillStateToDisk")]}
+                fieldType={FieldType.select}
+                fieldLabel={"I/O mode"}
```

Comment:
It shouldn't be done on FE, we should move this to BE, we already has options to do that. Let's talk